all/memory/replay_buffer.py

```diff
@@ -23,14 +23,17 @@ def update_priorities(self, indexes, td_errors):
 # Adapted from:
 # https://github.com/Shmuma/ptan/blob/master/ptan/experience.py
 class ExperienceReplayBuffer(ReplayBuffer):
-    def __init__(self, size, device=torch.device('cpu')):
+    def __init__(self, size, device='cpu', store_device='cpu'):
```

Comment:
Perhaps `store_device` could default to `None`, in which case `store_device` would just be set to `device`?

Reply:
Changed